fiftyone/core/service.py

```
@@ -249,7 +249,7 @@ def command(self):
 
         try:
             etau.ensure_dir(database_dir)
-        except:
+        except BaseException:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `BaseException`

(BLE001)